packages/framework/aqueduct/src/test/types/validateAqueductPrevious.generated.ts

```diff
@@ -40,26 +40,26 @@ use_old_ClassDeclaration_BaseContainerRuntimeFactory(
 /*
 * Validate forward compat by using old type in place of current type
 * If breaking change required, add in package.json under typeValidation.broken:
-* "ClassDeclaration_BaseContainerService": {"forwardCompat": false}
```

Comment:
You'll need to add these entries to package.json -- see some of the other removal PRs for examples.

Also make sure you add a note in BREAKING.md, under "2.0.0-internal.4.0.0 Breaking changes"